config.default

```diff
@@ -91,6 +91,12 @@
             "7.3"
           ]
         },
+        "astralinux": {
+          "base": deb,
```

Comment:
This will break the config file. Please wrap `deb` into `"`.

Reply:
fixed